### PR TITLE
PEO-1178 | Added i18n-tasks

### PR DESCRIPTION
**Pull request by WoLkErSs**
[PEO-1178](https://garage.atlassian.net/browse/PEO-1178)

### Description

Added i18n-tasks and removed unused locales

### Before submitting the merge request make sure the following are checked

- [x] Followed the style guidelines of this project
- [x] Performed a self-review of own code
- [ ] Wrote the tests that prove that fix is effective/that feature works

### CODE REVIEW

File: config/i18n-tasks.yml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+#   deepl_api_key: "48E92789-57A3-466A-9959-1A1A1A1A1A1A"
+
+## Do not consider these keys missing:
+# ignore_missing:
+# - 'errors.messages.{accepted,blank,invalid,too_short,too_long}'
+# - '{devise,simple_form}.*'
+
+## Consider these keys used:
+ignore_unused:
+  - '{errors}.*'
```
Comment (WoLkErSs) [inline config/i18n-tasks.yml]: Have ignored all `errors` becouse ignore by keys does not work

File: spec/concepts/api/v1/lib/operation/filtering_spec.rb [hunk 2]
```diff
@@ -106,9 +106,7 @@
     context 'when filter is not a hash' do
       let(:filtering_params) { 'not_a_hash' }
       let(:errors) { { filter: [I18n.t('errors.hash?')] } }
-      let(:error_localizations) { %w[errors.hash?] }
 
-      include_examples 'errors localizations'
```
Comment (chubchenko) [inline spec/concepts/api/v1/lib/operation/filtering_spec.rb:111]: Why did you remove it?

File: config/i18n-tasks.yml (hunk 1 above)
Comment (chubchenko) [reply config/i18n-tasks.yml]: Maybe instead of ignoring errors we can correctly set up a gem?

**Review by chubchenko — CHANGES_REQUESTED** [review]
Why did we removed all `errors localizations`?

File: spec/concepts/api/v1/lib/operation/filtering_spec.rb (hunk 2 above)
Comment (WoLkErSs) [reply spec/concepts/api/v1/lib/operation/filtering_spec.rb:111]: This test had checked locales wich stored in gems
I think we should not duplicate them

File: config/i18n-tasks.yml (hunk 1 above)
Comment (WoLkErSs) [reply config/i18n-tasks.yml]: Dry-validation implicitly used locales with text errors
Could you explain what should I set up?

File: spec/support/shared_examples/operation/errors_localizations.rb [hunk 3]
```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
-
-RSpec.shared_examples 'errors localizations' do
-  it 'errors localizations exist in i18n' do
-    expect(error_localizations).to be_defined_in_i18n
```
Comment (chubchenko) [inline spec/support/shared_examples/operation/errors_localizations.rb:5]: Do we need to remove `be_defined_in_i18n` matcher?

Comment (WoLkErSs) [reply spec/support/shared_examples/operation/errors_localizations.rb:5]: Think we should remove it becouse we use it nowhere

**Comment by chubchenko** [comment]
@WoLkErSs could you resolve the conflicts, thanks

**Review by chubchenko — APPROVED** [review]
